src/test/java/CalendarServletTest.java

```diff
@@ -40,17 +40,16 @@
 /** Test Calendar Servlet responds to client with correctly parsed Events. */
 @RunWith(JUnit4.class)
 public final class CalendarServletTest {
-  private AuthenticationVerifier authenticationVerifier;
-  private CalendarClientFactory calendarClientFactory;
-  private CalendarClient calendarClient;
-  private CalendarServlet servlet;
-  private HttpServletRequest request;
-  private HttpServletResponse response;
-  private StringWriter stringWriter;
-  private PrintWriter printWriter;
-  private final Gson gson = new Gson();
+  private static AuthenticationVerifier authenticationVerifier;
+  private static CalendarClientFactory calendarClientFactory;
+  private static CalendarClient calendarClient;
+  private static CalendarServlet servlet;
+  private static HttpServletRequest request;
+  private static HttpServletResponse response;
+  private static StringWriter stringWriter;
+  private static PrintWriter printWriter;
+  private static final Gson gson = new Gson();
```

Comment:
Nit: I'm sorry for the confusion. I meant only the gson field should be static. The others you are better off leaving non-static to avoid scenarios like the following possible sequence of calls performed by JUnit:
```
Test test1 = Test();
Test test2 = Test();
test1.init()
test2.init()
test1.test1()
# your static fields are not necessarily in the same state they were after the init() call
test2.test2()
```

(The gson is fine because it won't change throughout the tests.)